builder/proxmox/iso/config_test.go

```diff
@@ -42,11 +42,12 @@ func TestBasicExampleFromDocsIsValid(t *testing.T) {
       "ssh_username": "root",
       "ssh_timeout": "15m",
       "ssh_password": "packer",
-
+	
       "unmount_iso": true,
       "template_name": "fedora-29",
-      "template_description": "Fedora 29-1.2, generated on {{ isotime \"2006-01-02T15:04:05Z\" }}"
-    }
+      "template_description": "Fedora 29-1.2, generated on {{ isotime \"2006-01-02T15:04:05Z\" }}",
+	  "skip_convert_to_template": true
+	}
```

Comment:
The indentation is not consistent with the rest of the template here, the verbatim template being only indented with spaces.